controllers/etcdcluster_controller.go

```diff
@@ -83,14 +86,130 @@ func (r *EtcdClusterReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 			log.Error(err, "unable to create Service", "namespace", service.Namespace, "cluster", cluster.Name)
 			return ctrl.Result{}, err
 		}
+		log.V(1).Info("Created Service", "service name", service.Name)
 		return ctrl.Result{}, nil
 	}
+	log.V(2).Info("Service exists", "service name", service.Name)
+
+	peers := &etcdv1alpha1.EtcdPeerList{}
+	if err := r.List(ctx, peers, client.MatchingFields{"spec.clusterName": cluster.Name}); err != nil {
```

Comment:
Can we make `spec.clusterName` a const?

Reply:
Is a const in the function okay? Or do you want it pulled all the way out?

Reply:
I missed it was used in two places, I extracted this properly now.